.github/workflows/coverity.yml

```diff
@@ -0,0 +1,47 @@
+name: coverity
+
+on:
+  push:
+    branches: [ coverity_scan ]
+
+jobs:
+  scan:
+    runs-on: ubuntu-18.04
+    env:
+      TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
```

Comment:
Will work only after filling `secrets` of repository.

See more info [there](https://docs.github.com/en/actions/reference/encrypted-secrets).